client/src/app/styles/_buttons.scss

```diff
@@ -48,7 +48,7 @@
   font-size: $m-font;
   color: $text-normal;
   cursor: pointer;
-  padding: 12px 0 12px 0;
+  padding: $m-padding 0 $m-padding 0;
```

Comment:
isn't just `12px 0` enough?

Reply:
question still valid - from docs: "When two values are specified, the first padding applies to the top and bottom, the second to the left and right."